### ISSUE TITLE
Data-preprocess for Movinet on Kinetics600

### ISSUE THREAD

**Issue by Wangbk-dl** [tensorflow/models#11088]
# Prerequisites

Please answer the following question for yourself before submitting an issue.

- [ ] I checked to make sure that this issue has not been filed already.

## 1. The entire URL of the documentation with the issue

https://github.com/tensorflow/models/tree/master/official/...

## 2. Describe the issue

A clear and concise description of what needs to be changed.

I'm finetuning the movinet on Kinetics600 dataset. When I load the a0-stream checkpoint, choose 35 target classes that I'm interested and evaluate the precise of the loaded model, but I found that there was only 45.41% top-1 acc. I have not done any fintune yet. So I guess the result comes from the poor data pre-process. I didn't do anything special, but only normalize the pixels from 0-255 to 0-1 without any data-augmentation. 
I didn't find any instructions about how to augment the training data, so I came here for some help.
Anyone could help me with that?
